### PR TITLE
Migrate app to Sveltekit 1.0rc

### PR DESCRIPTION
**Pull request by petem-methods**
### What

Recreate app from scratch from latest Sveltekit scaffold.

### How to review

Check app still works as exptected.

### Who can review

Anyone

Notes
###

- Copied `quadsDataTileGrid.json` inside the vite root -- by default, Vite will not serve files outside the root path. Need to update script output location or allow this file to be served.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,69 +1,59 @@
 {
-  "name": "svelteapp",
+  "name": "dp-census-maps",
```
Comment (VivianAllen) [inline package.json:2]: I know this is confusing, but I think the apps name might be best as dp-census-atlas, unless we move everything again?

**Review by VivianAllen — APPROVED** [review]
very nice, all seems to work!

File: package.json (hunk 1 above)
Comment (petem-methods) [reply package.json]: Agreed. (This name isn't used for anything, as it's not a published packed but yeah, let's make it consistent.)

**Review by VivianAllen — APPROVED** [review]
✅

**Pull request merged**